### PR TITLE
mgr/dashboard: Fix CephPoolGrowthWarning alert

### PR DESCRIPTION
**Pull request by aaSharma14**
Prometheus reports an error - many-to-many matching not allowed: matching labels must be unique on one side for CephPoolGrowthWarning if we have same pool ids on two different instances.

Fixes: https://tracker.ceph.com/issues/58017
Signed-off-by: Aashish Sharma <aasharma@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by Pegonzal — APPROVED** [review]
Hey @aaSharma14 , the commit isn't signed

**Comment by epuertat** [comment]
> @aaSharma14 can you add a tracker for it? also could you explain in the description why the error happened and how the fix actually fixed the issue? (not clearly understanding since i've no clue in monitoring queries).

Agree, that's exactly the first thing that came to my mind: what's that "error" and, given it feels like a bug, what releases is this reproduced in?

File: monitoring/ceph-mixin/tests_alerts/test_alerts.yml [hunk 1]
```diff
@@ -1482,7 +1482,7 @@ tests:
       values: '1 1 1 1 1'
    promql_expr_test:
      - expr: |
-         (predict_linear(ceph_pool_percent_used[2d], 3600 * 24 * 5) * on(pool_id)
```
Comment (epuertat) [inline monitoring/ceph-mixin/tests_alerts/test_alerts.yml:1485]: And how's it possible that this test didn't catch the error? Is it not a PromQL error but a Grafana one?

Comment (epuertat) [reply monitoring/ceph-mixin/tests_alerts/test_alerts.yml:1485]: 👍🏼 thanks, @aaSharma14 !

**Comment by nizamial09** [comment]
jenkins retest this please

**Comment by nizamial09** [comment]
@aaSharma14 the signed-off-by check is failing. maybe its because you missed a space after the colon. 
`Signed-off-by:Aashish Sharma <aasharma@redhat.com>`

**Comment by aaSharma14** [comment]
jenkins test make check

**Comment by nizamial09** [comment]
jenkins test api

**Comment by nizamial09** [comment]
jenkins test dashboard

**Pull request merged**